parsers/jscript.c

```diff
@@ -1360,7 +1360,7 @@ static int parseMethodsInAnonymousObject (tokenInfo *const token)
 
 	tokenInfo *const anon_object = newToken ();
 	copyToken (anon_object, token, true);
-	anonGenerate (anon_object->string, "anonymousObject", JSTAG_VARIABLE);
+	anonGenerate (anon_object->string, "anonymousObject_", JSTAG_VARIABLE);
```

Comment:
Do we need this in this commit?

This commit is marked as "refactor". This change is not suitable for such a commit.
I expect if a commit is marked "refactor", most of all test cases are passed successfully without changing the test cases. (An exception is adding a new test case.)